examples/immersive-typography/complex/example.js

```diff
@@ -0,0 +1,180 @@
+let gold;
+let ring;
+let letter;
+
+let myFont;
+let randomx = [], randomy = [];
+let words = ["a", "b", "c", "d", "e"];
+
+let index = 0;
+let angle = 0;
+let a = 1;
+let x = 10;
+let speed = 0.5;
+
+let time = 0;
+let interval = 1000;
+
+let fontSize = 75;
+let hiArray;
+let samplefact = 0.25;
+
+function preload() {
+  createVRCanvas();
+  myFont = loadFont('../ITCAvantGardeStd-Demi.otf');
```

Comment:
This font isn't included in the PR. Also, I don't think that this is an open source font. If we are going to distribute the font file then it needs to be an opensource font or we need to load it from a resource that has the rights to distribute the font file.